packages/lesswrong/lib/vulcan-lib/fragments.ts

```diff
@@ -72,7 +72,9 @@ export const getDefaultFragmentText = <T extends DbObject>(collection: Collectio
     */
     const field: CollectionFieldSpecification<T> = schema[fieldName];
     // OpenCRUD backwards compatibility
-    return (field.resolveAs && !field.resolveAs.addOriginalField) || fieldName.includes('$') || fieldName.includes('.') || (options.onlyViewable && !(field.canRead || field.viewableBy));
+
+    const isResolverField = field.resolveAs && !field.resolveAs.addOriginalField && field.resolveAs.type !== "ContentType";
+    return isResolverField || fieldName.includes('$') || fieldName.includes('.') || (options.onlyViewable && !(field.canRead || field.viewableBy));
```

Comment:
This is still extremely hard to read but I appreciate that you've made it a bit easier to read 👍